### PR TITLE
Change legend labels and add notice

### PR DESCRIPTION
**Pull request by RubenSibon**

### CODE REVIEW

File: src/i18n/locales/nl/translations.json [hunk 1]
```diff
@@ -75,6 +75,11 @@
       "recommendedAndMandatoryRoutes": "Aanbevolen en verplichte routes",
       "displayOfRecommendedAndMandatoryRoutes": "Weergave van de aanbevolen en verplichte routes voor touringcars.",
```
Comment (manilde) [inline src/i18n/locales/nl/translations.json:76]: Aanpassing aangezien deze tekst altijd getoont wordt, niet alleen bij de agenda items 'aanbevolen' en 'verplichte', dus moet generieker
"recommendedAndMandatoryRoutes": "Situatie vanaf januari 2024",
      "displayOfRecommendedAndMandatoryRoutes": "Parkeerplaatsen, in- en uitstaphaltes en de verplichte touringcarroute in Amsterdam.",
"payAttentionToVerhicleHeights": "Let op de doorrijhoogtes.",

File: src/i18n/locales/nl/translations.json [hunk 2]
```diff
@@ -75,6 +75,11 @@
...
       "payAttentionToVerhicleHeights": "Let op de doorrijhoogtes.",
+      "_attention2024": {
+        "attention": "LET OP:",
+        "paragraph1": "Vanaf januari 2024 mogen touringcars zwaarder dan 7,5 ton niet meer binnen de S100 rijden. Uitzondering is de S100 zelf en de corridor Weesperstraat – Valkenburgerstraat – Kattenburgerstraat (aangegeven in lichtgroen).",
+        "paragraph2": "Voor de wegen aangegeven in het donkergroen geldt dat deze alléén toegankelijk zijn voor touringcars >7,5 ton mét ontheffing. Een ontheffing is alleen aan te vragen voor vervoer van specifieke doelgroepen (basisschoolleerlingen, mensen met een beperking, ouderen, artiesten) in combinatie met een bezoek aan een educatieve en/of culturele instelling."
+      },
```
[the remark this answers is not in the document]
Comment (RubenSibon) [reply src/i18n/locales/nl/translations.json:88]: This has been adapted as suggested.

File: src/i18n/locales/nl/translations.json (hunk 1 above)
Comment (RubenSibon) [reply src/i18n/locales/nl/translations.json]: Adapted as suggested

File: src/pages/Touringcar/components/MapLayers/__snapshots__/MapLayers.test.tsx.snap [hunk 3]
```diff
@@ -365,20 +365,69 @@ FAQ en contactgegevens"
...
+                        >
+                          <strong>
+                            Alleen met ontheffing
+                          </strong>
+                        </p>
+                        <p
+                          class="sc-jEACwC sc-gFqAkR jiqykX kBZfau"
+                          style="margin-block-end: .5em;"
+                        >
+                          Deze wegen zijn alléén toegankelijk voor touringcars &gt;7,5 ton mét ontheffing. Een ontheffing is alleen aan te vragen voor vervoer van specifieke doelgroepen (basisschoolleerlingen, mensen met een beperking, ouderen, artiesten) in combinatie met een bezoek aan een educatieve en/of culturele instelling.
```
Comment (thomasbrockmeier-ams) [inline src/pages/Touringcar/components/MapLayers/__snapshots__/MapLayers.test.tsx.snap:428]: Are the `>` characters from the source code displayed correctly with the conversion here to `&gt;`? #nitpick: the old texts seem to have had white space between the `>` and the `7.5`, whereas this does not. Perhaps consider double checking this to verify that this is consistent--provided that we care.

Comment (RubenSibon) [reply src/pages/Touringcar/components/MapLayers/__snapshots__/MapLayers.test.tsx.snap:428]: I can confirm that the `>` is correctly displayed in the app.

As far as textual consistency is concerned: that's a discussion to be had with the stakeholders who provide the content.

**Review by manilde — APPROVED** [review]

**Pull request merged**